language/stdlib/modules/sliding_nonce.move

```diff
@@ -0,0 +1,45 @@
+address 0x0 {
+module SlidingNonce {
+    use 0x0::Transaction;
+
+    resource struct T {
+        min_nonce: u64,
+        nonce_mask: u128,
+    }
+
+    // Aborts transaction if nonce can not be accepted
```

Comment:
Could you add a more detailed comment explaining when the nonce will/won't be accepted?